api/src/utils/media/csv/validation/csv-row-validator.ts

```diff
@@ -320,6 +330,10 @@ export const getValidFormatFieldsValidator = (config?: ColumnFormatValidatorConf
       }
 
       const rowValueForColumn = String(row[columnIndex]);
+
+      if (rowValueForColumn === 'undefined') {
```

Comment:
we have to check for the value equaling the actual string `undefined`?

Reply:
fixed it ... good catch.